at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -141,6 +145,12 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
     _syncComplete(syncRequest);
   }
 
+  SyncRequest _getSyncRequest() {
+    return _syncRequests.firstWhere(
```

Comment:
Lets have some Dart docs on what this logic is

Reply:
done